yarn.lock

```diff
@@ -4210,7 +4210,7 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react@*", "@types/react@>=16.9.0", "@types/react@^17.0.2":
+"@types/react@*":
```

Comment:
This can be coalesced with the type below it by deleting this entire entry and prepending `"types/react@*, ` to L4222, or by running `npx yarn-deduplicate --packages @types/react`.

Reply:
done ty